static/documentation/ROOMSTECHDESC-NOTE.txt

```diff
@@ -0,0 +1,5 @@
+The ROOMSTECHDESC.pdf document in tis directory was sourced from Interlisp.org Google Drive repository file
+medley-pdfs/rooms-pdfs/xd0e-RELEASE-rooms-2.0-users-src-ROOMSTECHDESC.PDF.
+
+FGH 2022-12-13
+
```

Comment:
should we have a loadup/build task of making these PDFs from their (TEdit? Notecards?) sources?

Reply:
For now, the Notecards doc is a scanned copy.  Haven't been able to recreate the right Notecards manual from TEdits.  Working on it,

Also for now, the ROOMS doc is a placeholder - this is not really the right doc for a ROOMS User Guide and/or primer.

Once we get legitimate TEdit-based docs straightened out, we should probably add the pdf generation to loadups.  But for now, probably just storing these docs in a static folder is the right thing to do.